published/external/xdp/details/afxdp.h

```diff
@@ -87,7 +87,7 @@ XskBind(
     _In_ XSK_BIND_FLAGS Flags
     )
 {
-    XSK_BIND_IN Bind = {};
+    XSK_BIND_IN Bind;
```

Comment:
These are intentionally being zero initialized. If this syntax isn't working, we should use [Rtl]ZeroMemory.